src/scss/utils/elemental.scss

```diff
@@ -0,0 +1,11 @@
+.element-holder {
+  margin: 3rem 0;
```

Comment:
Don't need to add top-margin to these blocks - just the bottom margin for spacing. So think you could change this to:
`margin: 0 0 3rem;`

Reply:
Done!